pyaccel/naff.py

```diff
@@ -64,4 +62,10 @@ def naff_general(signal, is_real=True, nr_ff=2, window=1):
     fourier = real + 1j*imag
     fourier = _np.squeeze(fourier)
     freqs = _np.squeeze(freqs)
+
+    if is_real:
+        freqs = _np.abs(freqs)
+        mask = freqs > 0.5
+        freqs[mask] = _np.abs(1-freqs[mask])
```

Comment:
as we talked in Teams, I think it is better to remove this `if` statement and change documentation.